frontend/src/components/header/header.scss

```diff
@@ -1,53 +1,70 @@
-@import "../../styles/variables.scss";
+@import '../../styles/variables.scss';
 
 $break-small: 320px;
 $break-medium: 0px;
 $break-large: 1200px;
 
+.main-header,
 .main-header-white {
-    background-color: $background;
-    height: 140px;
-    .text {
-        height: 25px;
-        width: 54px;
-        color: rgb(0, 0, 0);
-        font-family: Comfortaa;
-        font-size: 14px;
-        font-weight: bold;
-        letter-spacing: 1.1px;
-        line-height: 148px;
-        text-align: center;
-        text-decoration: none;
+  height: 140px;
+  .menu-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .menu {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    max-width: 600px;
+    padding: 0 30px;
+  }
+  .header-logo {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    img {
+      margin: 0;
+      height: 56px;
+      width: auto;
     }
-    .logo {
-        height: 56px;
-        vertical-align: center;
-        margin-top: 44px;
-        margin-left: 25%;
+  }
+  .text {
+    height: 25px;
+    width: 54px;
+    font-family: Comfortaa;
+    font-size: 14px;
+    font-weight: bold;
+    letter-spacing: 1.1px;
+    line-height: 148px;
+    text-align: center;
+    text-decoration: none;
+    &.selected {
+      padding-bottom: 10px;
+      border-bottom: 1.8px solid $violetCard;
     }
+  }
+  .logo {
+    height: 56px;
+    vertical-align: center;
+    margin-top: 44px;
+    margin-left: 25%;
```

Comment:
Avoid `%` units in margin and paddings since they'll change based on parent width.